tests/baselines/reference/user/npm.log

```diff
@@ -387,6 +387,8 @@ node_modules/npm/lib/install.js(948,8): error TS2454: Variable 'previousPath' is
 node_modules/npm/lib/install.js(985,53): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/install.js(1007,53): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/install/access-error.js(4,60): error TS2554: Expected 0-1 arguments, but got 2.
+node_modules/npm/lib/install/access-error.js(5,12): error TS2339: Property 'code' does not exist on type 'Error'.
```

Comment:
I'm not sure why we didn't issue an error here before. It *might* be because of the erroneous call on the previous line, but the type is reported as `Error`, not `any`.